### PR TITLE
Add kubestash v2024.12.9 release note

### PR DESCRIPTION
**Pull request by ishtiaqhimel**

### CODE REVIEW

File: content/post/kubestash-v2024.12.9/index.md [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+Here we will give an example of query verification strategy for KubeDB managed `MySQL`.
+
+At first, we need to check if the verification function is present in our cluster using the following command:
+
+```bash
+$ kubectl get functions | grep -i `kubedbverifier`
+kubedbverifier            19h
+```
+
+Assuming that a KubeDB managed `MySQL` instance with a configured backup already exists. Now, we need to create a `MySQL` instance for verification in our preferred namespace (i.e. `verify`). Here is an example of `MySQL` instance for verification:
```
Comment (hmsayem) [inline content/post/kubestash-v2024.12.9/index.md:39]: To verify the backup of a MySQL database, a separate MySQL instance  is required to run the verification process for the original database.  Here is an example of a MySQL instance for verification:

File: content/post/kubestash-v2024.12.9/index.md [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+    accessModes:
+      - ReadWriteOnce
+    resources:
+      requests:
+        storage: 50Mi
+```
+
+Here, the configuration for the verification application should be similar to the backup application as we are going to restore the backup data in the verification application.
+
+Here is the example of `BackupVerifier` that verifies the `MySQL`:
```
Comment (hmsayem) [inline content/post/kubestash-v2024.12.9/index.md:61]: is it mandatory to have the same configuration?

File: content/post/kubestash-v2024.12.9/index.md [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+        table: user
+        rowCount:
+          operator: Equal
+          value: 100
+      - database: store
+        table: order
+  sessionHistoryLimit: 2
+```
+
+Now we need to update `BackupConfiguration` to refer the `BackupVerifier`.
```
Comment (hmsayem) [inline content/post/kubestash-v2024.12.9/index.md:98]: Here is the BackupConfiguration configured for the original database, which refers to the BackupVerifier

File: content/post/kubestash-v2024.12.9/index.md [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+  action:
+    exec:
+      command:
+      - curl
+      - -X
+      - POST
+      - -H
+      - "Content-Type: application/json"
+      - -d
+      - `{"text":":x: Name: {{ .Name }} Namespace: {{.Namespace}} Phase: {{.Status.Phase}}"}`
```
Comment (hmsayem) [inline content/post/kubestash-v2024.12.9/index.md:171]: fix the text that turns into an icon in the site.

**Review by hmsayem — CHANGES_REQUESTED** [review]

File: content/post/kubestash-v2024.12.9/index.md [hunk 5]
```diff
@@ -0,0 +1,195 @@
...
+    pod:
+      selector: name=test-app, test=hook
+```
+
+
+### Improvements & Bug Fixes
+
+- We've fixed an issue for external databases backup. The database version was not resolved during backup which is fixed within this release.
+- We've fixed an issue where addon job template was not resolved in the `BackupConfiguration`. That has been fixed in this release.
+
```
Comment (hmsayem) [inline content/post/kubestash-v2024.12.9/index.md:184]: - We’ve fixed an issue with external database backups where the database version was not resolved during the backup process. This has now been fixed in this release.
- We have also fixed an issue where the addon job template specified in the BackupConfiguration was not applied properly. It has been fixed in this release.

File: content/post/kubestash-v2024.12.9/index.md (hunk 2 above)
Comment (ishtiaqhimel) [reply content/post/kubestash-v2024.12.9/index.md:61]: What if the user configures it with a different version?

File: content/post/kubestash-v2024.12.9/index.md [hunk 6]
```diff
@@ -0,0 +1,195 @@
...
+  storageType: Durable
+  storage:
+    accessModes:
+      - ReadWriteOnce
+    resources:
+      requests:
+        storage: 50Mi
+```
+
+Here, the configuration for the verification application should be similar to the backup application as we are going to restore the backup data in the verification application.
```
Comment (hmsayem) [inline content/post/kubestash-v2024.12.9/index.md:59]: It should have a similar configuration to the original backup application  (e.g., the database version should match), as we are going to restore the original application's backup data into it.

**Review by hmsayem — CHANGES_REQUESTED** [review]

**Review by anisurrahman75 — APPROVED** [review]

**Review by hmsayem — APPROVED** [review]

**Pull request merged**